src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -33,6 +33,7 @@
 from ...exceptions import RegistrationFailed
 from ...registry import register
 from ...utils import execute_unless_result_exists
+from ..objects_api.submission_registration import prepare_data_for_registration
```

Comment:
Hum, this would then to be applied to some zgw stuff imported in the objects api plugin module as well:
https://github.com/open-formulieren/open-forms/blob/7b31de3ff4bdb85aefc94d06280d1f69949b8ede/src/openforms/registrations/contrib/objects_api/plugin.py#L10-L16

Reply:
no, because it's using the shared functionality in `openforms.contrib.zgw`, not using private API from `openforms.registrations.contrib.zgw_apis` :)